keps/sig-node/3953-node-resource-hot-plug/kep.yaml

```diff
@@ -0,0 +1,50 @@
+title: Node Resource Hot Plug
+kep-number: 3953
+authors:
+  - "@Karthik-K-N"
+  - "@mkumatag"
+  - "@kishen-v"
+owning-sig: sig-node
+participating-sigs:
+  - sig-node
+status: provisional
+creation-date: 2023-10-04
+reviewers:
+  - "@smarterclayton"
+  - "@ffromani"
+  - "@SergeyKanzhelev"
+  - "@haircommander"
+  - "@tallclair"
+approvers:
```

Comment:
neither of listed people can approve the KEP. It should be one of @mrunalp @dchen1107 or @derekwaynecarr